MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionPresenter.cpp

```diff
@@ -265,6 +271,11 @@ void EnggDiffractionPresenter::processShutDown() {
   cleanup();
 }
 
+void EnggDiffractionPresenter::processPlotRepChange() {
+  m_view->saveSettings();
```

Comment:
It seems to produce [error](http://builds.mantidproject.org/job/pull_requests-win7/3060/console) if I do not include the function processPlotChange().. I might have pasted the previous function by accident. Because this function is required here, should I leave an empty function? @FedeMPouzols